Assessment/CustomerCommand/Console/Command/ImportCustomerCommand.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Assessment\CustomerCommand\Console\Command;
+
+use Assessment\CustomerCommand\Api\ProfileInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use \Magento\Framework\Console;
+
+/**
+ * Class ImportCustomerCommand
+ *
+ * @package Assessment\CustomerCommand\Console\Command
+ */
+class ImportCustomerCommand extends Command
+{
+    private const NAME = 'name';
+    protected $profiles;
+    const COMMAND_NAME = 'customer:importer';
```

Comment:
```suggestion
    private const NAME = 'name';
    protected $profiles;
    const CUSTOMER_COMMAND_NAME = 'customer:importer';
```
Naming could have been better...

Reply:
changed the names of the variable